openapi/components/schemas/QuoteCreateOrder.yaml

```
@@ -0,0 +1,468 @@
+type: object
+title: QuoteCreateOrder
+required:
+  - websiteId
+  - customerId
+  - items
+  - action
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
+    type: string
+    readOnly: true
+    enum:
+      - subscription-order
+      - one-time-order
+  action:
+    description: |-
+      Action of the quote.
+      On accept quote new order is created. `orderId` and `invoiceId` are filled with new generated values from the order.
+    type: string
+    enum:
+      - create
+    example: create
+  acceptanceConditions:
+    type: array
+    writeOnly: true
+    description: If conditions are fulfilled the quote is automatically accepted.
+    items:
+      type: string
+      enum:
+        - customer
+        - signature
+        - payment
+        - partial-payment
+      x-enumDescriptions:
+        customer: Quote can be accepted by customer.
+        signature: Quote can be accepted if merchant received the signed quote from the customer.
+        payment: Related invoice must be paid in full.
+        partial-payment: Related invoice must be paid in full or partially paid.
+    example: ['customer', 'payment']
+    default: ['customer']
+  acceptanceFulfillment:
+    type: array
+    readOnly: true
+    description: Conditions with fulfillment flag.
```

suggestion: to add more detail
I found this description difficult to understand (I'm not sure if my suggestion changes the meaning)

```suggestion
    description: Conditions that must be met to fulfill the order. 
```